### ISSUE TITLE
Loss Function 문제 

### ISSUE THREAD

**Issue by insurgent92** [DeepBaksuVision/YOLOv1#7]
![image](https://user-images.githubusercontent.com/15168540/45095806-f24ae380-b159-11e8-9b6a-9dad98401544.png)

Loss Function에서 width와 height와 관련되서 sqrt가 들어가야 할 것 같은데 현재 코드에서는 sqrt를 취하지 않습니다. 
https://github.com/DeepBaksuVision/YOLOv1/blob/edf7a06f681c3a53d6fd088bcd2d43ebf030d9bf/pytorch_version/yolov1.py#L269

하지만 sqrt를 취하게되면 negative sign으로 nan이 뜹니다. 

darknet yolo 같은 경우 다음과 같은 전략을 취하고 있습니다.

https://github.com/pjreddie/darknet/blob/680d3bde1924c8ee2d1c1dea54d3e56a05ca9a26/src/detection_layer.c#L170

**Comment by insurgent92** [comment]
Ground Truth에만 sqrt를 취하고 prediction 값은 sqrt를 안취하면 되는군요. 우선 나온 해법입니다.
